src/olympia/amo/urls.py

```diff
@@ -17,6 +17,11 @@
 
 api_patterns = [
     re_path(r'^site/$', views.SiteStatusView.as_view(), name='amo-site-status'),
+    # This duplicates services patterns under /api/vX/services/, which is
+    # useful to reach those services from an API request, to make them work
+    # with services.a.m.o. or test their behavior with request.is_api being
+    # True.
+    re_path(r'^services/', include(services_patterns)),
```

Comment:
some of the patterns - the 403, 403, 500 views - are not JSON so should they really be exposed under /api?  (😬)

Reply:
`404` and `500` are. Only `403` is not. I could implement a quick one.

Reply:
44ce9f42fbd206bdf7341d8b1d744e41e17b573f adds docs, limits to the views we care about, adds 403 API support, and tests.